<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialGui</class>
 <widget class="QMainWindow" name="SerialGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port Setup</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbStartDataRead">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Communication</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopDataRead">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>149</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Communication</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>101</width>
      <height>266</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Port Comm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPortComm"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Baud Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbBaudRate"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Data Bits</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbDataBits"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Parity Bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbParityBit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Flow Control</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbFlowControl"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Stop Bit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbStopBit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Serial Dialog Window</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="txDialogWindow">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pbSendData">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>149</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>81</y>
      <width>67</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Transfer Data</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="etTransferData">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>147</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <zorder>pbSendData</zorder>
   <zorder>label_7</zorder>
   <zorder>etTransferData</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pbStartDataRead</zorder>
   <zorder>StopDataRead</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuK">
    <property name="title">
     <string>Serial Port</string>
    </property>
   </widget>
   <addaction name="menuK"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
